lib/galaxy/tools/actions/__init__.py

```diff
@@ -949,7 +949,7 @@ def filter_output(tool, output, incoming):
             if not eval(filter.text.strip(), globals(), incoming):
                 return True  # do not create this dataset
         except Exception as e:
-            log.debug('Tool %s output %s: dataset output filter (%s) failed: %s' % (tool.id, output.name, filter.text, e))
+            log.info(f'Tool {tool.id} output {output.name}: dataset output filter ({filter.text}) failed: {e}')
```

Comment:
This should be at the debug level, that's going to spam the logs even if everything is alright. The log level for this module needs to be changed in planemo.

Reply:
Alright, then I revert this. Can you tell me an example when this is triggered and you do not want to see the message?

Reply:
Would this be done here: https://github.com/galaxyproject/planemo/blob/f5b2a4aa2d0f93c1152adddde0ac61f037fe64ad/planemo/galaxy/config.py#L130?

Reply:
That's the rough idea, would be awesome if we can tune it there.

Reply:
Maybe like so: https://github.com/galaxyproject/planemo/pull/1165